### PR TITLE
Add functionality to alpha graph

### PR DESCRIPTION
**Pull request by nids2001**
Use shift modifier to delete points
Click and drag points to edit (needs work)

### CODE REVIEW

**Comment by dcwhite** [comment]
Another TODO: older networks stored their alpha points without scaling the x,y values; we could try adding some code to scale these correctly.

**Comment by dcwhite** [comment]
@nids2001 Merge commits from https://github.com/SCIInstitute/SCIRun/tree/colormap-preview-state-fixes and we can test this with just the remove feature done. Oh, and update the UI file to explain how to remove points with shift-click (add a new label underneath or something).

File: src/Interface/Modules/Visualization/CreateStandardColorMap.ui [hunk 1]
```diff
@@ -45,6 +45,16 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QLabel" name="label_7">
+        <property name="text">
+         <string>Hold shift to remove a point</string>
```
Comment (allywarner) [inline src/Interface/Modules/Visualization/CreateStandardColorMap.ui:51]: Either have a period here or remove the period from the sentence above about alphas.

**Comment by allywarner** [comment]
Click and drag is not implemented, right?

**Comment by dcwhite** [comment]
Correct, only point removal

**Pull request merged**